firebase-test/firebasetest.js

```diff
@@ -33,3 +48,45 @@ dbUsername.on("value", function(dataSnapshot) {
 	usernameBox.textContent = dataSnapshot.val();
 	console.log( dataSnapshot.val() );
 });
+
+// Event Listener for Login
+// Use Firebase with Github Auth to log in the user
+firebase.auth().signInWithRedirect(provider).catch(function(error){
+	// Log any errors to the console
+	console.log(error);
+});
+
+
+// Event Listener for Logout
+// Change your logout button's event listener 
+firebase.auth().signOut().catch(function(error){
```

Comment:
Same issue as on line 54 -- I don't see an event listener here! See my comment on line 54 for this section too. Same questions apply here: ***when*** should the user be logged out?